operator/builtin/transformer/add/config_test.go

```diff
@@ -0,0 +1,148 @@
+package add
+
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+
+	"github.com/open-telemetry/opentelemetry-log-collection/entry"
+	"github.com/open-telemetry/opentelemetry-log-collection/operator/helper"
+)
+
+type configTestCase struct {
+	name   string
+	expect *AddOperatorConfig
+}
+
+func TestMoveGoldenConfig(t *testing.T) {
```

Comment:
```suggestion
func TestAddGoldenConfig(t *testing.T) {
```

Reply:
Or just `TestGoldenConfig`, since it's the only one in this package. If you do this, please also update `move` accordingly.